scripts/coder-dev.sh

```diff
@@ -17,7 +17,7 @@ fi
 
 if [[ ! -x "${CODER_DEV_BIN}" ]]; then
 	echo "Run this command first:"
-	echo "go build -o ${CODER_DEV_BIN} ${PROJECT_ROOT}/cmd/coder"
+	echo "go build -o ${CODER_DEV_BIN} ${PROJECT_ROOT}/enterprise/cmd/coder"
```

Comment:
Can we add an `--agpl` flag to this script as well for folks who need to easily hack on the OSS version?

Reply:
This change just controls what is printed, and the script is meant to wrap `coder` and pass args thru, so I don't think an `--agpl` flag is appropriate here.  But, I do agree for `develop.sh`